src/main/config/feature.h

```diff
@@ -54,6 +54,7 @@ typedef enum {
     FEATURE_ESC_SENSOR = 1 << 27,
     FEATURE_ANTI_GRAVITY = 1 << 28,
     FEATURE_DYNAMIC_FILTER = 1 << 29,
+    FEATURE_OSD_PROFILE = 1 << 30
```

Comment:
I don't see why this should be a feature, if the adjustment range is never assigned then the default (0) page should always be displayed.

Reply:
You're right, it's not necessary and yes better that it isn't a feature.  Now removed.